package/package.json

```diff
@@ -8,7 +8,7 @@
   "main": "digitalmarketplace/all.js",
   "sass": "digitalmarketplace/all.scss",
   "engines": {
-    "node": "10.19.0"
+    "node": ">= 4.2.0"
```

Comment:
do we really support any node version from 4.2.0 upwards?

Reply:
Hmmmm. Just tried running `npm install` with node 4.2.0 and got a truly amazing number of warnings.

Maybe `>= 10.19.0` would be more realistic?

Reply:
@lfdebrux I would guess anything 10.0+ has the same API.